accord-core/src/test/java/accord/impl/basic/InMemoryJournal.java

```diff
@@ -497,6 +501,9 @@ private Diff(int flags, CommandUpdate update)
                     case EXECUTES_AT_LEAST:
                         changes.put(EXECUTES_AT_LEAST, after.executesAtLeast());
                         break;
+                    case MIN_UNIQUE_HLC:
```

Comment:
If we already store HLC in waitingOn, do we need to also save it separately?

Reply:
We aren’t, I’ve stopped WaitingOn from persisting either executesAtLeast or UniqueHLC. This also means an update to one of these components only requires updating that component. The WaitingOn provider accepts them as inputs now.